src/components/Calculator.js

```diff
@@ -14,64 +15,67 @@ export default function Calculator() {
 
   return (
     <div className="calculator-div">
-      <Display value={value} />
-      <button type="button" onClick={handleClick} name="AC">
-        AC
-      </button>
-      <button type="button" onClick={handleClick} name=" +/-">
-        +/-
-      </button>
-      <button type="button" onClick={handleClick} name="%">
-        %
-      </button>
-      <button type="button" className="orange" onClick={handleClick} name="÷">
-        ÷
-      </button>
-      <button type="button" onClick={handleClick} name="7">
-        7
-      </button>
-      <button type="button" onClick={handleClick} name="8">
-        8
-      </button>
-      <button type="button" onClick={handleClick} name="9">
-        9
-      </button>
-      <button type="button" className="orange" onClick={handleClick} name="x">
-        x
-      </button>
-      <button type="button" onClick={handleClick} name="4">
-        4
-      </button>
-      <button type="button" onClick={handleClick} name="5">
-        5
-      </button>
-      <button type="button" onClick={handleClick} name="6">
-        6
-      </button>
-      <button type="button" className="orange" onClick={handleClick} name="-">
-        -
-      </button>
-      <button type="button" onClick={handleClick} name="1">
-        1
-      </button>
-      <button type="button" onClick={handleClick} name="2">
-        2
-      </button>
-      <button type="button" onClick={handleClick} name="3">
-        3
-      </button>
-      <button type="button" className="orange" onClick={handleClick} name="+">
-        +
-      </button>
-      <button type="button" className="span-two" onClick={handleClick} name="0">
-        0
-      </button>
-      <button type="button" onClick={handleClick} name=".">
-        .
-      </button>
-      <button type="button" className="orange" onClick={handleClick} name="=">
-        =
-      </button>
+      <Quote />
```

Comment:
- [ ] Kindly i recommend you to import  the <Quotes> component into the **main component(App.j**s), instead of the on the component for calculator.  Kindly fix on App.js just like this : 
```
import Calculator from './components/Calculator';
import Quote from './Quote';
function App() {
  return (
    <div className="App">
      <h1> Math Magicians</h1>
      <Calculator />
      <Quote />
    </div>
  );
}

export default App;
```